hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/snapshot/TestOMSnapshotCreateRequest.java

```diff
@@ -311,9 +318,10 @@ public void testEntryExists() throws Exception {
         omResponse.getStatus());
   }
 
-  private void renameKey(String fromKey, String toKey) throws IOException {
-    OmKeyInfo toKeyInfo = addKey(fromKey);
-    OmKeyInfo fromKeyInfo = addKey(toKey);
+  private void renameKey(String fromKey, String toKey, long offset)
+      throws IOException {
+    OmKeyInfo toKeyInfo = addKey(fromKey, offset + 1L);
```

Comment:
This need to be 
OmKeyInfo toKeyInfo = addKey(toKey, offset + 1L);
OmKeyInfo fromKeyInfo = addKey(fromKey, offset + 2L); ?